File: scripts/update-shop-name.ts
import { PrismaClient } from '@prisma/client';

const prisma = new PrismaClient();

async function main() {
  await prisma.shopSettings.update({
    where: { key: 'shop_name' },
    data: { value: 'Билэг хүргэлт' }
  });
  console.log('Shop name updated');
}

main().catch(console.error).finally(() => prisma.$disconnect());

File: scripts/update_shop_name.py
import psycopg2

conn = psycopg2.connect('postgresql://postgres:postgres@localhost:5432/krono')
cursor = conn.cursor()
cursor.execute('UPDATE "ShopSettings" SET "value" = %s WHERE "key" = %s', ("Онлайн дэлгүүр", "shop_name"))
conn.commit()

cursor.execute('SELECT * FROM "ShopSettings" WHERE "key" = %s', ("shop_name",))
print(cursor.fetchall())
